### ISSUE TITLE
createGroup returns Undefined on success

### ISSUE THREAD

**Issue by pfavaloro** [Cimpress/coam-client#8]
If I do 

`let id = await CoamClient.createGroup(jwt, groupName,  description);`

I can see that the group gets created in COAM and the Promise resolves, but `id` is undefined. I would expect either the groupId or the entire response entity be returned.

Looking at the code, it seems like it should return the entire response body, so I'm not sure where the bug is.

Thanks!

**Comment by pfavaloro** [comment]
I can provide a little more context. The function looks like this if I look at the source downloaded from npm 
```
{
    key: "createGroup",
    value: function createGroup(name, description) {
      var data = this.__config({
        url: '/auth/access-management/v1/groups',
        method: 'POST',
        data: {
          name: name,
          description: description
        }
      }); // [{user_id / name / email}]


      return this.__exec(data).then(function (p) {
        return p.canonical_principals;
      });
    }
  }

```
Simply by removing the .then... after `this.__exec`, it works as expected. 

```
{
    key: "createGroup",
    value: function createGroup(name, description) {
      var data = this.__config({
        url: '/auth/access-management/v1/groups',
        method: 'POST',
        data: {
          name: name,
          description: description
        }
      }); // [{user_id / name / email}]


      return this.__exec(data);
    }
  }
```

Apologies for the ignorance on why this code is different than what is in this repo, not exactly sure how that works.

**Comment by istanishev** [comment]
Thank you @pfavaloro ! This should be addressed in v0.1.7 ;)
